src/web/app/pages-session/session-result-page/session-result-page.component.html

```diff
@@ -1,19 +1,35 @@
 <h1>
   Feedback Session Results
 </h1>
-<div class="card bg-light top-padded" *ngIf="session">
-  <div class="card-body row" style="padding: 40px;">
-    <label class="col-sm-2 control-label">Course ID:</label>
-    <div class="col-sm-10">{{ session.courseId }}</div>
+<div class="card card-plain" ngtif="session">
```

Comment:
Should be `*ngIf`. Please spend some time to read up on this [concept](https://angular.io/api/common/NgIf) :)